modules/ROOT/pages/csae.adoc

```diff
@@ -0,0 +1,88 @@
+= Getting started with Teradata VantageCloud ClearScape Analytics^TM^
+:experimental:
+:page-author: Vidhan Bhonsle
+:page-email: vidhan.bhonsle@teradata.com
+:page-revdate: January 10th, 2024
+:description: Getting started with Teradata ClearScape Analytics Experience
+:keywords: data warehouses, compute storage separation, teradata, vantage, cloud data platform, business intelligence, enterprise analytics, jupyter, teradatasql, ipython-sql, clearscape, csae
+
+== Overview
+
+https://www.teradata.com/platform/clearscape-analytics[ClearScape Analytics^TM^] is a powerful analytics engine in https://www.teradata.com/platform/vantagecloud[Teradata VantageCloud]. It delivers breakthrough performance, value, and growth across the enterprise with the most powerful, open and connected AI/ML capabilities on the market.
+
+image::VantageCloud.png[VantageCloud,align="center",width=50%]
+
+ClearScape Analytics Experience enables user to configure free lightweight cloud environments where they can scale AI/ML quickly and effectively to solve the most complex challenges, reduce cost and accelerate time to value.  
+
+In this how-to we will go through the steps for creating environment in Teradata VantageCloud ClearScape Analytics Experience and access in-built demos.
+
+== Create a ClearScape Analytics Experience account
+
+Head over to https://www.teradata.com/getting-started/demos/clearscape-analytics[ClearScape Analytics Experience] and create a free account.
+
+image::csae_register.png[Register,align="center",width=75%]
+
+Sign in to your https://clearscape.teradata.com/sign-in[ClearScape Analytics account] to create an environment and access demos.
+
+image::csae_signin.png[Sign in,align="center",width=60%]
+
+== Create an Environment
+
+Once signed in, click on *CREATE ENVIRONMENT*
+
+image::csae_create_env.png[Create environment,align="center",width=75%]
+
+You will need to provide:
+
+[cols="1,1"]
+|====
+| *Variable* | *Value*
+
+| *environment name* 
+| A contextual name like "demo"
+
+| *database password* 
+| A password of your choice
+
+| *Region* 
+| Select one from the dropdown +
+* us-central +
+* us-east +
+* us-west +
+* southamerica-east +
+* europe-west +
+* asia-south +
+* asia-northeast +
+* asia-southeast +
+* australia-southeast
+
+|====
+
+IMPORTANT: Note down database password for future usage of environment 
+
+image::csae_env_params.png[Environment params,align="center",width=75%]
+
+Click on *CREATE* button to complete the creation of your environment and now, you can see details of your environment.
+
+image::csae_env_details.png[Environment details,align="center",width=75%]
+
+== Access demos
+
+The Teradata VantageCloud ClearScape Analytics environment comes with a bunch of pre-built demos to help you with your use-case. +
```

Comment:
The Teradata VantageCloud ClearScape Analytics platform includes a variety of ready-made demos designed to assist you in addressing your specific needs.

Reply:
```suggestion
The ClearScape Analytics Experience environment includes a variety of demos that showcase how to use analytics so solve business problems across many industries. +
```